### ISSUE TITLE
SWIFT4 recognition only numbers

### ISSUE THREAD

**Issue by AndreaDiFrancia** [gali8/Tesseract-OCR-iOS#388]
Is it possible to recognize only the numbers within an image?

**Comment by AndreaDiFrancia** [comment]
Yes, I tried it, I have this configuration, I'll attach it below, but it takes literal characters and turns them into numerical ones, you could help me. Thank you



if let tesseract = G8Tesseract(language:"eng+ita") {

            tesseract.delegate = self



            returnImage { (image) in

               // let imageRender = image.g8_grayScale()

                //tesseract.engineMode = .tesseractOnly

                //tesseract.pageSegmentationMode = .singleLine

                tesseract.charWhitelist = "01234567890"

                // tesseract.charBlacklist = "abcdefghilmnopqrstuvzABCDEFGHILMNOPQRSTUVZ"

                //tesseract.maximumRecognitionTime = 120.0

                tesseract.image = image.g8_blackAndWhite()

                tesseract.recognize()

                self.text?.text = tesseract.recognizedText

                return

            }

this configuration doesn't work
________________________________
Da: Oleg <notifications@github.com>
Inviato: martedì 13 novembre 2018 14:28
A: gali8/Tesseract-OCR-iOS
Cc: Andrea Di Francia; Author
Oggetto: Re: [gali8/Tesseract-OCR-iOS] SWIFT4 recognition only numbers (#388)


Yes, it is. Use charWhitelist property.

**Comment by oleghnidets** [comment]
Try `engineMode` as `tesseractCubeCombined` or `cubeOnly`.

Also read this https://stackoverflow.com/a/53281416/4869734

**Comment by AndreaDiFrancia** [comment]
doesn't work, I try in multiple way

________________________________
Da: Oleg <notifications@github.com>
Inviato: mercoledì 14 novembre 2018 17:13
A: gali8/Tesseract-OCR-iOS
Cc: Andrea Di Francia; Author
Oggetto: Re: [gali8/Tesseract-OCR-iOS] SWIFT4 recognition only numbers (#388)


Try engineMode as tesseractCubeCombined or cubeOnly.

Also read this https://stackoverflow.com/a/53281416/4869734

**Comment by IOSwift2019** [comment]
camera image can't get actual output
gujarati font not get.

**Comment by burcukutluay** [comment]
it does not work, I tried every way. Is there anybody who find the solution?
